README.md

```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
 
 The following subprojects are owned by the Working Group:
 
-{{
-
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Organization Infrastructure
+  * Description: Repositories to manage the working group itself - these are not products for users
+  * Repositories:
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
+    * https://github.com/RobotWebTools/rwt_dockers
```

Comment:
Is this being used for anything? Is it a demo for rwt usage?

Reply:
It was to build a docker for demo server. But, unmaintained for a long time. It can be removed.